tasks/main.yml

```diff
@@ -1,33 +1,33 @@
 ---
 - name: Import OS-specific variables.
-  include_vars: '{{ item }}'
+  ansible.builtin.include_vars: '{{ item }}'
   with_first_found:
     - '{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml'
     - '{{ ansible_distribution }}.yml'
     - '{{ ansible_os_family }}.yml'
 
 - name: Ensure keepalived and its dependencies are installed.
-  package:
+  ansible.builtin.package:
     name: "{{ keepalived_packages }}"
     state: present
-  when: keepalived_install
+  when: keepalived_install | bool
```

Comment:
I think we should change the defautl value:
defaults/main.yml:keepalived_install: no
=> defaults/main.yml:keepalived_install: false
Because in documentation it is said it's a boolean.

We can still keep the `| bool` to avoid breaking changes for those who use `keepalived_install: yes`